rest/route/stats.go

```diff
@@ -331,21 +328,29 @@ func (tsh *testStatsHandler) Parse(ctx context.Context, r *http.Request) error {
 	project := gimlet.GetVars(r)["project_id"]
 	vals := r.URL.Query()
 
-	if vals.Get("presto") == "true" {
-		return tsh.parsePrestoStatsFilter(project, vals)
-	}
-
-	tsh.filter = stats.StatsFilter{Project: project}
-	err := tsh.StatsHandler.parseStatsFilter(vals)
+	identifier, err := dbModel.GetIdentifierForProject(project)
 	if err != nil {
-		return errors.Wrap(err, "invalid query parameters")
+		return gimlet.ErrorResponse{
+			Message:    fmt.Sprintf("getting project identifier for '%s'", project),
+			StatusCode: http.StatusInternalServerError,
+		}
 	}
-	err = tsh.filter.ValidateForTests()
-	if err != nil {
-		return errors.Wrap(err, "invalid filter")
+
+	// If this project is owned by Server and uses Resmoke, we need to use
+	// Evergreen test stats.
```

Comment:
This is for future-proofing, in subsequent tickets I am going to get rid of the `CedarTestResultsEnabled` field entirely so new projects are all sent to Cedar test results and for Mongo Resmoke projects we will send to both Evg and Cedar until we can get rid of this route (PM-2940).

Reply:
I don't know if its worth all the extra piping of a new flag since we are going to delete all this code soon with PM-2940. The mongosync projects are now definitely going to Evergreen (triple checked my DB update haha).

Reply:
got it. But if there's a one-to-one mapping between the projects that are `CedarTestResultsEnabled` and the ones that we want to fetch the stats from evergreen could we just repurpose that flag for this? Does it _have_ to be removed?